src/data/ACTIONS/layers/patch5.5.ts

```diff
@@ -0,0 +1,9 @@
+import {Layer} from 'data/layer'
+import {ActionRoot} from '../root'
+
+// tslint:disable:no-magic-numbers
```

Comment:
don't need this, we don't even have tslint any more. can you nuke it in one of your other prs?

Reply:
Yeah no worries